### PR TITLE
Update versions - 2019-10-10

### PR DESCRIPTION
**Pull request by ovk6**

### CODE REVIEW

**Comment by armandomiani** [comment]
@googlebot I consent

File: templates/Dockerfile.template [hunk 1]
```diff
@@ -48,13 +50,15 @@ RUN git clone https://github.com/prometheus/memcached_exporter /memcached_export
...
-RUN cd /memcached_exporter && \
-			export PATH=$PATH:/usr/local/go/bin && \
-			git checkout 1454051534ec5b9c65201907b4c90cabbd862185 && \
-			memcached -u memcache -d && \
-      make
+RUN cd /memcached_exporter \
+	&& export PATH=$PATH:/usr/local/go/bin \
+	&& git checkout {{ $template.exporterShaCommit }} \
+	&& memcached -o modern -u memcache -d \
+    && make precheck style check_license lint unused common-build
```
Comment (ovk6) [inline templates/Dockerfile.template]: Fix indent

Comment (armandomiani) [reply templates/Dockerfile.template]: Solved.

File: versions.yaml [hunk 2]
```diff
@@ -30,15 +30,19 @@ versions:
...
-      - '1.5.16'
+      - '1.5.19'
       - '1.5'
       - '1'
       - 'latest'
     from: *from
+    templateArgs:
+      # Reference commit in this repository:
+      # https://github.com/prometheus/memcached_exporter/
+      exporterShaCommit: d7eadc3523ec9731b7865a188093f70140d54a8d
```
[the remark this answers is not in the document]
Comment (armandomiani) [reply versions.yaml]: Fixed.

**Pull request merged**